lib/spack/env/cc

```diff
@@ -191,11 +202,11 @@ while [ -n "$1" ]; do
         -Xlinker,*)
```

Comment:
Here is another bug (apparently mine).  `-Xlinker` doesn't use commas.  So we should probably remove the comma here and make the Xlinker handling less fancy.  It looks like [from here](http://stackoverflow.com/questions/7221141/any-difference-between-wl-option-and-xlinker-option-syntax-for-gcc) and from googling that Xlinker won't ever be used with a comma, and it *always* has a space after it.  Can you change this part?